tests/integration/cli/repos_test.py

```diff
@@ -0,0 +1,72 @@
+from os import path
+from shutil import rmtree
+from subprocess import check_call
+from tempfile import mkdtemp
+
+import git
+
+from dusty import constants
+from dusty.log import log_to_client
+from ...fixtures import busybox_single_app_bundle_fixture
+from ...testcases import DustyIntegrationTestCase
+
+
+class TestReposCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestReposCLI, self).setUp()
+        busybox_single_app_bundle_fixture(num_bundles=1)
+        self.run_command('bundles activate busyboxa')
+        self.temp_repos_dir = mkdtemp()
+        self.old_repos_dir = constants.REPOS_DIR
+        constants.REPOS_DIR = self.temp_repos_dir
```

Comment:
Usually you don't want to set values of module variables like this.  I feel like we have other code in tests that you could look at for accomplishing something similar